### PR TITLE
[CROSSDATA-473][Driver] Provide a method to check whether the cluster is alive

### PR DESCRIPTION
**Pull request by darroyocazorla**
Once the PR is approved, API docs must be updated

### CODE REVIEW

**Comment by miguel0afd** [comment]
Nice work!

**Comment by pfcoperez** [comment]
+1

File: driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala [hunk 1]
```diff
@@ -285,6 +289,42 @@ class Driver private(private[crossdata] val driverConf: DriverConf,
...
+  /**
+   * Gets the addresses of servers up and running
+   *
+   * @since 1.3
+   * @return A sequence with members of the cluster ready to serve requests
+   */
+  def serversUp(): Future[Seq[Address]] = {
+    import collection.JavaConverters._
+    clusterState().map{ cState =>
+      cState.getMembers.asScala.filter( _.status == MemberStatus.Up).map(_.address).toSeq
```
Comment (pfcoperez) [inline driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala]: filter + map = collect
filter + map = flatMap

Comment (darroyocazorla) [inline driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala]: Replaced with:
cState.getMembers.asScala.collect {
   case member if member.status == MemberStatus.Up => member.address
}.toSeq

**Pull request merged**